cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -337,9 +337,9 @@ public CoachMarkBuilder setOnDismissListener(OnDismissListener listener) {
          * Set which animation will be used when displaying/hiding the coach mark
          * 
          * @param animationStyle
-         *      the resource ID of the animation to be shown
+         *      the resource ID of the Style to be used for showing and hiding the coach mark
          */
-        public CoachMarkBuilder setAnimation(@AnimRes int animationStyle) {
+        public CoachMarkBuilder setAnimation(@StyleRes int animationStyle) {
```

Comment:
Is this effectively an API change? Can we still pass @AnimRes s without warnings?